CONTRIBUTING.md

```diff
@@ -250,6 +250,26 @@ We already have many config options, and we should avoid fixing an issue by addi
 - has workaround using existing options
 - can be addressed with a plugin instead
 
+## Release
+
+If you have publish access, the steps below explains how to cut a release for a package. There are two phases for the release step: "Release" and "Publish".
+
+"Release" is done locally to generate the changelogs and git tags:
+
+1. Make sure the git remote for https://github.com/vitejs/vite is set as `origin`.
+2. In the `vite` project root `main` branch, run `git pull` and `pnpm i` to get it up-to-date.
+3. Run `pnpm release` and follow the prompts to cut a release for a package. It will generate the changelog, a git release tag, and push them to `origin`. You can run with the `--dry-run` flag to test it out.
```

Comment:
```suggestion
3. Run `pnpm release` and follow the prompts to cut a release for a package. It will generate the changelog, a git release tag, and push them to `origin`. You can run with the `--dry` flag to test it out.
```